clowdapp.yaml

```diff
@@ -80,19 +80,99 @@ objects:
           failureThreshold: 3
           httpGet:
             path: /
-            port: 8000
+            port: 9000
```

Comment:
With this change are we running both processors on port 9000 instead of existing 8000?

Reply:
No, these ports are for the livenessProbe earlier we used to rely on the [monitoring thread](https://github.com/RedHatInsights/ros-backend/blob/d234b4094adb9192152d4801ef1477b6731a0a2a/ros/processor/main.py#L33) to keep track of running processor and if the http server in monitoring thread return http status code 500 then openshift used to automatically restart the pods. Now we are taking the different approach, every pod will have Prometheus server running and now livenessProbe will curl that Prometheus server for checking if application is alive/running.